src/components/layout/responsive-row.tsx

```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
   ${({ theme }) =>
     theme.screens.sm
       ? css`
-          flex-direction: column;
+          flex-flow: column;
           align-items: start;
         `
       : css`
-          flex-direction: row;
+          flex-flow: row;
+          align-items: center;
```

Comment:
the design doesn't require it to be aligned in center

Reply:
This is not required but it was useful in this case👇. And one thing to note adding that didn't changed other things :)
![image](https://user-images.githubusercontent.com/46858011/89638253-20e31300-d8c9-11ea-80dd-34b26252a9eb.png)
![image](https://user-images.githubusercontent.com/46858011/89638328-3e17e180-d8c9-11ea-9380-0b0907a766d1.png)